### PR TITLE
Limbsnakes properly disember limbs

### PR DESCRIPTION
**Pull request by eeSPee**
## About The Pull Request

Now removes gloves and handcuffs

## Why It's Good For The Game

For some reason, it doesn't update your limbs/status properly properly.

## Changelog
:cl:
fix: limbsnakes properly disember limbs
tweak: Chimeracost doubled
tweak: biodegrade cost halved
/:cl:

### CODE REVIEW

**Comment by february-ops** [comment]
Bruh this just makes the freedom thing from lings useless now

**Comment by Isy232** [comment]
> Bruh this just makes the freedom thing from lings useless now

This does not affect lockers. Weld a ling into a locker and limbsnake dont help.

**Comment by Sinmero** [comment]
Lings already have Biodegrade to get rid of handcuffs. Snakes cost 1 point and also summons snakes that will attack people vs Biodegrade 2 points whose sole purpose is to remove handcuffs. This change will make Biodegrade useless. You could make snakes cost 2 and biodegrade 1.

**Comment by eeSPee** [comment]
actually not a bad idea

File: code/modules/antagonists/changeling/powers/biodegrade.dm [hunk 1]
```diff
@@ -4,8 +4,7 @@
 	helptext = "This is obvious to nearby people, and can destroy standard restraints and closets."
 	button_icon_state = "biodegrade"
 	chemical_cost = 30 //High cost to prevent spam
-	dna_cost = 2
-	req_human = 1
+	dna_cost = 1
```
Comment (PowerfulBacon) [inline code/modules/antagonists/changeling/powers/biodegrade.dm:7]: this does require human

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**